test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/ElementalValueProviderTests.cs

```diff
@@ -14,7 +14,7 @@ public class ElementalValueProviderTest
         [InlineData("MyProperty", "MyProperty")]
         [InlineData("MyProperty.SubProperty", "MyProperty")]
         [InlineData("MyProperty[0]", "MyProperty")]
-        public async Task ContainsPrefixAsync_ReturnsTrue_IfElementNameStartsWithPrefix(string elementName, 
+        public void ContainsPrefix_ReturnsTrue_IfElementNameStartsWithPrefix(string elementName, 
                                                                                         string prefix)
```

Comment:
Shortened method definition messing up the old #YOLO wrapping. Not horrendous of course but please take a quick cleanup pass through the updated code.